src/perl/vdotest/VDOTest/Vdorecover.pm

```diff
@@ -228,18 +231,26 @@ sub testFullFilesystem {
   # Sleep to give the script time to start up.
   sleep(20);
 
-  # Remount the filesystem to clear it's errors.
-  $fs->unmount();
+  # We should now be sitting at the prompt from vdorecover (for lvmvdo device).
+  # Remount the filesystem, delete some data, and do a fstrim.
+  # Logged dmesg output should show start and completion of recovery.
   $fs->mount();
-
-  # Delete some data and do a fstrim.
   $dir2->rm(async => 0);
-  my $machine = $self->getDevice()->getMachine();
   $machine->assertExecuteCommand("sudo fstrim " . $fs->getMountDir());
   $self->syncDeviceIgnoringErrors();
+  $device->runOnHost("sudo dmesg -c", 1);
+  $fs->unmount();
+  $device->runOnHost("sudo dmesg -c > /dev/null");
```

Comment:
Please remove the space between `>` and `/dev/null`.  It should be `>/dev/null`

Reply:
Fixed in the latest update to the PR.